packages/editor-ui/src/components/canvas/elements/edges/CanvasEdge.vue

```diff
@@ -20,6 +35,35 @@ const path = computed(() =>
 		targetPosition: props.targetPosition,
 	}),
 );
+
+const connection = computed<Connection>(() => ({
+	source: props.source,
+	target: props.target,
+	sourceHandle: props.sourceHandleId,
+	targetHandle: props.targetHandleId,
+}));
+
+onMounted(() => {
+	document.addEventListener('keydown', onKeyDown);
+});
+
+onUnmounted(() => {
+	document.removeEventListener('keydown', onKeyDown);
+});
+
```

Comment:
I'm a bit worried about adding event listeners for each edge, as there could potentially be hundreds of them. Couldn't we delegate these from some parent container component?

Reply:
Agree, also I guess bulk delete would not work if each one handles `del` key individually

Reply:
Thank you! That was a lack of judgement from my side. I've moved the keydown handler to `Canvas.vue`.